cmd/tools/migration/backend/etcd210.go

```diff
@@ -293,6 +295,35 @@ func (b etcd210) loadLastDDLRecords() (meta.LastDDLRecords, error) {
 	return records, nil
 }
 
+func (b etcd210) loadLoadInfos() (meta.CollectionLoadInfo210, error) {
+	loadInfo := make(meta.CollectionLoadInfo210)
+	_, collectionValues, err := b.txn.LoadWithPrefix(legacy.CollectionMetaPrefixV1)
+	if err != nil {
+		return nil, err
+	}
+	for _, value := range collectionValues {
+		collectionInfo := querypb.CollectionInfo{}
+		err = proto.Unmarshal([]byte(value), &collectionInfo)
+		if err != nil {
+			return nil, err
+		}
+		if collectionInfo.InMemoryPercentage < 100 {
```

Comment:
Why we ignore these unloaded collections? One quick question, if the users called the `load` before query service is down, should querycoord recover these collections anyway?

Reply:
since v2.2, the loading collection will be released after restart